app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -68,8 +68,43 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class EndExperiment(graphene.Mutation):
+    nimbus_experiment = graphene.Field(NimbusExperimentType)
+    message = ObjectField()
+    status = graphene.Int()
+
+    class Arguments:
+        input = ExperimentIdInput(required=True)
+
+    @classmethod
+    def mutate(cls, root, info, input: ExperimentIdInput):
+        experiment = NimbusExperiment.objects.get(id=input.id)
+
+        if experiment.status == NimbusExperiment.Status.LIVE:
+            experiment.is_end_requested = True
+            experiment.save()
```

Comment:
We probably want to add a changelog here to capture the user that requested the end.